recipes/bond/all/test_package/example.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+#include <bond/core/bond_const_enum.h>
+
+int main() {
+    std::cout << "bond is installed successfully" << std::endl;
```

Comment:
Please call at least one bond function (not using a define or header-only function).
That way we can be sure the library is tested properly.